src/RuntimeSettings.php

```diff
@@ -64,6 +71,11 @@ public function isDev()
      */
     public function getValue($name, $default = null)
     {
+        $url = $this->getUrlParam($name);
```

Comment:
I don't understand how that would improve things. Each type of input has its own quirks - $_GET needs lowercased key, environment uses uppercased, and header is uppercased + HTTP_X_ + replacing - with _.